### DISCUSSION TITLE
Looking for a REPUTABLE company

### DISCUSSION THREAD

**Discussion by kcabral817** [MarimerLLC/csla#3727] [Questions]
I am looking for a reputable software solution provider that knows Azure app services inside and out for a reasonable price.  I need to have the site looked at from a deployment, _**security**_ and appropriate services.  I have to foot the bill myself so I need a company that can work within and be available to upscale as I bring on more clients--ie. take over management.

**Comment by rockfordlhotka** [comment]
I don't know how you define "reasonable price", but I do work with Xebia and that's a key focus area for the [Xebia Microsoft Services](https://xebia.com/digital-transformation/microsoft-services/) group.

**Comment by rockfordlhotka** [comment]
@kcabral817 I think you've been hacked - or at least need to change your password. This is the second time in 2 weeks I've had to delete a post that appears to be from you, but includes a spam link.

**Comment by kcabral817** [comment]
Check
